src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -166,6 +167,10 @@ const EventEntries = createReactClass({
         {!utils.objectIsEmpty(event.device) && (
           <EventDevice group={group} event={event} />
         )}
+        {!isShare &&
+          features.has('event-attachments') && (
```

Comment:
No change required because of existing code, but there's a new `Feature` component: